### DISCUSSION TITLE
Versioning of Docker images

### DISCUSSION THREAD

**Discussion by DennisLundtoft** [Azure/ipam#148] [Q&A] [answered]
Hi,

Browsing through the Wiki and GitHub documentation I cannot seem to find anything regarding versioning of docker images and the code in general.

Is there any other versions other than latest? And if not, can we expect no breaking changes on it?

**Comment by DCMattyG** [comment, marked as answer]
Hey there @DennisLundtoft, there are two answers to your question....

1. At this moment in time, there isn't versioning. That said, all of the changes to this point have been additional features (e.g. non-breaking)
2. I'm currently working on GitHub Actions to automate the end-to-end testing of the code & API's before they are pushed to production (this is very manual today). Part of this automation will also include automated versioning, so that is indeed coming very soon.

So yes, versioning will be here in the near future, and there aren't currently any updates planned that will result in a breaking change. I know that isn't a *perfect* answer, but it's an *honest* answer 😉
